ethcore/src/client/config.rs

```diff
@@ -125,6 +125,9 @@ pub struct ClientConfig {
 	pub max_round_blocks_to_import: usize,
 	/// Snapshot configuration
 	pub snapshot: SnapshotConfiguration,
+	/// used by the RPC, returns an error if block information was requested
+	/// and there are gaps in the db
+	pub allow_empty_block_result: bool
```

Comment:
I think this should be added to `EthClientOptions`, this way we don't need to expose `ClientConfig`.